src/System.Threading.Tasks/tests/TaskScheduler/TaskSchedulerTests.cs

```diff
@@ -291,6 +291,7 @@ public static void RunSynchronizationContextTaskSchedulerTests_Negative()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Uses reflection to access an internal method of the TaskScheduler class.")]
```

Comment:
Does this mean that it will never be supported in uapaot? If it will eventually, I would rather have this marked as Active Issue and link it to the pending work.